app/data/schema/schema-v1.json

```diff
@@ -424,6 +425,20 @@
                 },
                 "routing_rules": {
                   "ref": "#/definitions/routing_rules"
+                },
+                "skip_condition": {
```

Comment:
Agree. was just raising this approach to see if people agreed before i refactored and wrote tests etc.